package.json

```diff
@@ -36,10 +35,10 @@
   },
   "devDependencies": {
     "coveralls": "^2.11.6",
-    "nyc": "^5.6.0",
+    "nyc": "^6.2.1",
     "rcinfo": "^0.1.3",
     "run-waterfall": "^1.1.1",
-    "standard": "^5.4.1",
+    "standard": "^6.0.8",
```

Comment:
Strictly speaking, it is an unrelated change, so, sorry, I will remove and address in a separate PR.